lib/sisimai/data.rb

```diff
@@ -392,8 +392,8 @@ def self.make(data: nil, **argvs)
 
           if o.deliverystatus.empty?
             # Set pseudo status code
-            textasargv = sprintf('%s %s', o.replycode, p['diagnosticcode'])
-            textasargv = textasargv.gsub(/\A[ ]/, '')
+            textasargv = o.replycode + ' ' + p['diagnosticcode'])
```

Comment:
__[Reek]__

[Syntax] This file has unexpected token tRPAREN


<!-- Comment by [SideCI](https://sideci.com) -->